waspc/cli/Command/Common.hs

```diff
@@ -0,0 +1,38 @@
+module Command.Common
+    ( findWaspProjectRootFromCwd
+    , findWaspProjectRoot
+    ) where
+
+import System.Directory (getCurrentDirectory, doesPathExist, doesFileExist)
+import qualified System.FilePath as FP
+import Data.Maybe (fromJust)
+import Control.Monad.Except (throwError)
+import Control.Monad.IO.Class (liftIO)
+import Control.Monad (when, unless)
+
+import StrongPath (Path, Abs, Dir)
+import qualified StrongPath as SP
+import Command (Command, CommandError(..))
+import Common (WaspProjectDir, dotWaspRootFileInWaspProjectDir)
+
+
+findWaspProjectRoot :: Path Abs (Dir ()) -> Command (Path Abs (Dir WaspProjectDir))
+findWaspProjectRoot currentDir = do
+    let absCurrentDirFp = SP.toFilePath currentDir
```

Comment:
```suggestion
    let absCurrentDir = SP.toFilePath currentDir
```
If you didn't have Fp suffix before, why now?

Reply:
aha FP and SP is how you differentiate between path and strong path

Reply:
then better call it wp, weakPath :D Fp sounds general

Reply:
Ah yes well hm. I use FP for FilePath. I don't have a better idea really. wp nobody knows what is :D.